### PR TITLE
Items created/updated to correct tech writer's build

### PR DESCRIPTION
**Pull request by deb-intel**
Signed-off-by: Deb Taylor <deb.taylor@intel.com>

### CODE REVIEW

**Comment by deb-intel** [comment]
These updates are to help my build to not return so many errors when I do make html. I had help from @intelkevinputnam

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,6 @@ _build
 *.sav
 *.log
 *.warnings
+.tox
```
Comment (intelkevinputnam) [inline .gitignore:7]: Looks like this is in here 2x. Safe to remove one.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(name='Sphinx GUI Utility',
+      version='0.1',
+      description='Build Sphinx docs from a GUI',
+      author='Kevin Putnam',
+      author_email='kevin.putnam@intel.com',
+      url='https://gitlab.devtools.intel.com/kputnam/doc-gui/',
```
Comment (intelkevinputnam) [inline setup.py:10]: Change the author, email, and URL to match the project.

File: scripts/requirements.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
-breathe>=4.9.1
-sphinx>=1.7.5
-docutils>=0.14
+breathe==4.14.1
+sphinx==2.4.4
+docutils==0.16
```
Comment (marc-hb) [inline scripts/requirements.txt:3]: These seem to be "bleeding edge" versions. Except for PR #225 I could build everything just fine with my stock sphinx 2.1.2 

Versions aside, @deb-intel can we please revert to `>=` to force fewer users to have one virtualenv per sphinx project?

Cc: @mmaka1 , @xiulipan, @lgirdwood .

Comment (marc-hb) [reply scripts/requirements.txt:3]: @deb-intel , @intelkevinputnam you forgot to update the documentation:
https://thesofproject.github.io/latest/contribute/process/docbuild.html#install-documentation-tools

> Our documentation processing has been tested to run with:
> 
> Python 3.6.3
> Doxygen version 1.8.13
> Sphinx version 1.7.5
> Breathe version 4.9.1
> docutils version 0.14
> sphinx_rtd_theme version 0.4.0
> 

( A more general discussion about versioning happened in #198, thanks again @intelkevinputnam  )

**Pull request merged**